test/config.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+RESOURCE_ID = {
+    "test-resource-1": "rbccps.org/e096b3abef24b99383d9bd28e9b8c89cfd50be0b/example.com/test-category/test-resource-1",
+    "test-resource-2": "rbccps.org/e096b3abef24b99383d9bd28e9b8c89cfd50be0b/example.com/test-category/test-resource-2",
+    "test-resource-3": "rbccps.org/e096b3abef24b99383d9bd28e9b8c89cfd50be0b/example.com/test-category/test-resource-3",
+    "test-resource-4": "rbccps.org/e096b3abef24b99383d9bd28e9b8c89cfd50be0b/example.com/test-category/test-resource-4",
+    "public": "rbccps.org/e096b3abef24b99383d9bd28e9b8c89cfd50be0b/example.com/test-category/test-resource.public"
+}
+
+YOUTUBE_URL = ['https://www.youtube.com/watch?v=iDO9J_3OVJ0',
+               'https://www.youtube.com/watch?v=wKWldDnCZQ0',
+               'https://www.youtube.com/watch?v=QXOXIMgHgZ0',
+               'https://www.youtube.com/watch?v=R3GfuzLMPkA'
+               ]
+
+VIDEOS = {
+    "countdown" : 'iDO9J_3OVJ0',
+    "Asha" : 'wKWldDnCZQ0',
+    "Artpark" : 'QXOXIMgHgZ0',
+    "HD" : 'R3GfuzLMPkA'
+}
+
+RTMP_HLS = 'rtmp+hls'
+RTMP = 'rtmp'
+PUSH_VALID = 'push_valid'
+PUSH_INVALID = 'push_invalid'
+PLAY_VALID = 'play_valid'
+PLAY_INVALID = 'play_invalid'
+FRAME_RATE = 'frame_rate'
+
+RECORD_SRC_DIR = '../nginx/record/'
+
+ACL_SET_ENDPOINT = 'https://localhost:8443/auth/v1/acl/set'
```

Comment:
This should use the ``AUTH_SERVER`` environment variable instead of using ``localhost:8443``